### ISSUE TITLE
Rename modular content to linked items

### ISSUE THREAD

**Issue by petrsvihlik** [Kentico/cloud-sdk-swift#27]
Based on our vision regarding content items, we have to rename "Modular content element" to "Linked items element". From the Delivery SDK's point of view, it means that we are going to rename everything that contains "modularContent" to "linkedItems".

Checklist:
- [ ] updated all members (methods/properties/classes/...)
- [ ] updated .MD files
- [ ] updated tests
- [ ] updated github wiki

Keep in mind, that the Delivery API responses will remain the same. They'll still contain "modular_content".

Example implementation: https://github.com/Kentico/delivery-sdk-net/pull/130

Plan your release so that the changes go live on 2018-10-03 and onwards.

**Comment by IpshitaC** [comment]
Hi @petrsvihlik 
Can I take this up as a part of Hacktoberfest?

**Comment by petrsvihlik** [comment]
certainly, @IpshitaC. I've just sent you a collaborator invitation. if you accept it, i'll be able to assign the issue to you.

**Comment by IpshitaC** [comment]
I've accepted the invite! 🎉
